scripts/release.sh

```
@@ -69,6 +69,122 @@ if [[ "$VIEWER_PERMISSION" != "ADMIN" && "$VIEWER_PERMISSION" != "WRITE" ]]; the
+            ;;
+        "get_repository_description")
+            gh repo view --json description,topics,language -q '"Description: " + (.description // "No description") + "\nLanguage: " + (.language // "Unknown") + "\nTopics: " + (.topics | join(", "))' 2>/dev/null || echo "Could not fetch repository description"
+            ;;
+        "get_previous_releases")
+            local limit
+            limit=$(echo "$arguments" | jq -r '.limit // 5')
+            gh release list --limit "$limit" --json name,tagName,body,publishedAt -q '.[] | "Release: " + .name + " (" + .tagName + ")\nPublished: " + .publishedAt + "\nNotes:\n" + (.body // "No release notes") + "\n---"' 2>/dev/null || echo "No previous releases found"
+            ;;
+        "get_project_structure")
+            local depth
+            depth=$(echo "$arguments" | jq -r '.depth // 2')
```

Shellcheck (0.10.0)

[warning] depth appears unused. Verify use (or export if used externally).

(SC2034)